### ISSUE TITLE
Pytorch implemenation

### ISSUE THREAD

**Issue by ptran1203** [TachibanaYoshino/AnimeGAN#51]
Hi @TachibanaYoshino,

Great work! Your project is really interesting, so I decided to implement the algorithm in Pytorch based on your Tensorflow code.

I tried to make the architecture of G and D to be the same as yours, and train on the same config (`loss weights`) but the result is a little bit different. So I tuned the `loss weights` to get the best result.

Currently, my results look like `anime`, but some generation images still have some artifacts. Maybe I will need to train longer.

Could you add this implementation into your README.md, I think It helps other people when looking for a Pytorch version :smile:

thank you very much.

my repo: https://github.com/ptran1203/pytorch-animeGAN

Some of my results:

![1 (46)_anime](https://user-images.githubusercontent.com/36766404/119455213-94c0b000-bd63-11eb-88b8-8f8a3e8dea08.jpg)
![1 (36)_anime](https://user-images.githubusercontent.com/36766404/119455166-87a3c100-bd63-11eb-93be-4fbe012530f0.jpg)

**Comment by TachibanaYoshino** [comment]
Ok, I'll add it to the readme later.

**Comment by ptran1203** [comment]
Thank you!
